src/enum_bitset.h

```diff
@@ -70,6 +71,18 @@ class enum_bitset
             return get_pos( enum_traits<E>::last );
         }
 
+        void serialize( JsonOut &json ) const {
+            json.start_array();
+
+            for( int i = 0; i < static_cast<int>( size() ); i++ ) {
```

Comment:
```suggestion
            for( size_t i = 0; i < size(); i++ ) {
```
Saves a `static_cast`, and I think it should work.